cmd/account/servicequotas/common.go

```diff
@@ -0,0 +1,36 @@
+package servicequotas
+
+import (
+	"errors"
+
+	awsv1alpha1 "github.com/openshift/aws-account-operator/pkg/apis/aws/v1alpha1"
+)
+
+/*GetSupportedRegions returns a []string of regions supported
+return []string of any regions found
+	if allRegions: return all found regions, ignore filter
+	if !allRegions:
+		filter set: return 1 element if region matches
+		filter == "": return all found regions
+return error if no regions are returned
+*/
```

Comment:
```suggestion
```

Reply:
Its a pseudo-code comment describing how the function behaves.
You did ask to understand what the function did.

Reply:
Comment changed as requested.